lab2.md

```diff
@@ -54,13 +54,13 @@ You'll see an ASCII logo generated like this:
 | | | | | | | |/ / 
 | | |_| | |_| / /_ 
 |_|\___/ \___/_/(_)
-``` 
+```
 
 ## Learn about the CLI
 
 You can now test out the CLI, but first a note on alternate gateways URLs:
 
-If your gateway is not deployed at http://127.0.0.1:8080 then you will need to specify the `--gateway` flag followed by the alternate URL such as http://127.0.0.1:8080/. 
+If your gateway is not deployed at http://127.0.0.1:8080 then you will need to specify the `--gateway` flag followed by the alternate URL.
```

Comment:
Since we changed all the `localhost` to `127.0.0.1` this phrase sounded strage. I deleted the last part.